doc/userguide/rules/ldap-keywords.rst

```diff
@@ -163,3 +163,155 @@ Example of a signature that would alert if a packet has more than 2 LDAP respons
 .. container:: example-rule
 
   alert ip any any -> any any (msg:"Packet has more than 2 LDAP responses"; :example-rule-emphasis:`ldap.responses.count:>2;` sid:1;)
+
+ldap.responses.result_code
+--------------------------
+
+Suricata has a ``ldap.responses.result_code`` keyword that can be used in signatures to identify
+and filter network packets based on LDAP result code.
+
+Syntax::
+
+ ldap.responses.result_code: code[,index];
+
+ldap.responses.result_code uses :ref:`unsigned 32-bit integer <rules-integer-keywords>`.
+
+This keyword maps to the following eve fields:
+``ldap.responses[].bind_response.result_code``
+``ldap.responses[].search_result_done.result_code``
+``ldap.responses[].modify_response.result_code``
+``ldap.responses[].add_response.result_code``
+``ldap.responses[].del_response.result_code``
+``ldap.responses[].mod_dn_response.result_code``
+``ldap.responses[].compare_response.result_code``
+``ldap.responses[].extended_response.result_code``
```

Comment:
This requires a formatting to actually be rendered as a list (same applies to the `error_message` list):
```suggestion
This keyword maps to the following eve fields:

   - ``ldap.responses[].bind_response.result_code``
   - ``ldap.responses[].search_result_done.result_code``
   - ``ldap.responses[].modify_response.result_code``
   - ``ldap.responses[].add_response.result_code``
   - ``ldap.responses[].del_response.result_code``
   - ``ldap.responses[].mod_dn_response.result_code``
   - ``ldap.responses[].compare_response.result_code``
   - ``ldap.responses[].extended_response.result_code``
```